Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -6,6 +6,8 @@ import UIKit
 final class ManagePledgeViewController: UIViewController {
   // MARK: - Properties
 
+  private let viewModel: ManagePledgeViewModelType = ManagePledgeViewModel()
```

Comment:
I've moved the `viewModel` to the top of Properties section. This is consistent with the rest of the codebase and makes it easier to glance at the same place where we would in other VC